lua/autorun/server/sv_daily_restart.lua

```diff
@@ -151,16 +213,31 @@ local function waitUntilRestartHour()
     local minutesOffset = 60 - currentMinute - 1
 
     -- We are this many seconds into the hour
-    local secondsAndMinutes = secondsOffset + minutesToSeconds( minutesOffset )
+    local secondsAndMinutes = secondsOffset + ( minutesOffset * 60 )
 
-    local secondsToWait = (hoursLeft * 3600) - secondsAndMinutes
+    local secondsToWait = (hoursLeft * SECONDS_IN_MINUTE) - secondsAndMinutes
 
     local timeToRestart = currentTime() + secondsToWait
     sendRestartTimeToClients( timeToRestart )
+end
 
-    timer.Create( DailyRestartTimerName, secondsToWait, 1, onAlertTimeout )
+local function createRestartTimer(seconds)
```

Comment:
Would this need to be ran in `waitUntilRestartHour` too? The countdown for the restart is never being initialized.